scripts/write_deployinst_input.py

```diff
@@ -188,66 +187,124 @@ def read_template(template):
     return output_template
 
 
-def reactor_render(list, template, mox_template, output_file):
+def refine_name(name_data):
+    """ Takes the name data and decodes and refines it.
+
+    Parameters
+    ----------
+    name_data: str
+        reactor name data from csv file
+
+    Returns
+    -------
+    name: str
+        refined and decoded name of reactor
+    """
+
+    name = name_data.decode('utf-8')
+    start = name.find('(')
+    end = name.find(')')
+    if start != -1 and end != -1:
+        name = name[:start]
+    return name
+
+def reactor_render(data_list, output_file, is_cyborg=False):
```

Comment:
I recommend that every function return something. In this case, you should return the reactor_body object.

Reply:
I'm not quite sure if I understand. Would it just return the last `reactor_body` object then?

Reply:
yes